andela/AndelaEatsUI
# #160410646 Implement admin meals page (UI)

#### What does this PR do?
- Adds view page for Admins to view all meals added on the platform.

#### Description of Task to be completed?
- Add meals icon and link to sidebar
- Add meals react component
- Install react-testing-library
- Write test suites for meals component

#### How should this be manually tested?
- Log in to the app
- Switch to the admin dashboard from the user profile drop-down menu or [here](http://andelaeats-dev.andela.com:3000/admin/meals)
- You should see a static page displaying meals

#### What are the relevant pivotal tracker stories?
- [#160410646](https://www.pivotaltracker.com/story/show/160410646)

#### Background Context
* This is a UI only task, the functionality will be added in a separate story

#### Screenshots
<img width="1440" alt="screen shot 2018-09-12 at 12 09 27 am" src="https://user-images.githubusercontent.com/32123313/45392959-a1d3fa00-b621-11e8-9e8a-b424b61c101e.png">
